portal/models/qb_status.py

```diff
@@ -510,6 +510,8 @@ def patient_research_study_status(patient, ignore_QB_status=False):
          have been met, such as assigned clinician if applicable.
      - ready: set True or False based on complex rules.  True means pending
          work user can immediately do.  NOT determined w/ ``ignore_QB_status``
+     - staff_eligible: if false, staff controls should be disabled.
```

Comment:
@pbugni  my only concern is the name of this variable and others in this PR... this is not limited to the staff role. It is for any role that can complete the intervention, which is I believe staff, staff_admin, and clinician. Maybe use `intervention_qnr_available`?

Reply:
👍